src/background/services/keyAutoAdd.ts

```diff
@@ -38,10 +39,11 @@ export class KeyAutoAddService {
   constructor({
     browser,
     storage,
+    appName,
     browserName,
     t,
-  }: Pick<Cradle, 'browser' | 'storage' | 'browserName' | 't'>) {
-    Object.assign(this, { browser, storage, browserName, t });
+  }: Pick<Cradle, 'browser' | 'storage' | 'appName' | 'browserName' | 't'>) {
```

Comment:
No need to use `Pick` since whole cradle is passed by default when using `PROXY_MODE`.
```suggestion
  }: Cradle) {
```

Reply:
Would've to manually pass each param when instantiating `KeyAutoAdd` (not using awilix there, it's not on container).

Reply:
🤦